windows/win_host.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2014, Phil Schwartz <schwartzmx@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+DOCUMENTATION = '''
+---
+module: win_host
+version_added: ""
+short_description: Hostname setting, and domain unjoining/joining module.
+description:
+     - Change the host's hostname, and domain.  Allows active-directory domain joining/unjoining, along with workgroup joining/unjoining. Uses Powershell's Add-Computer and Remove-Computer.
+options:
+  hostname:
+    description:
+      - Hostname to change to, or a comma separated list of computers to add to the domain
+    required: no
+    default: Current Host Name
+    aliases: []
```

Comment:
I know this is months ago now but ... there is an example of aliases in win_file.ps1 L25-37 as you can dest or name as aliases for path (although looking at the code now the error if none of them are specified is 'missing required argument: path'.